src/components/IssueType.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+import { ErrorMessage } from "formik";
+import AutoCompletTypeField from './AutocompleteTypeField';
+const IssueType = ({ rest, items, handleAddressChange, handleAddressSelect,  pageFieldName }) => {
+
+	return (
+		<React.Fragment>
+
+			<div>
+				<label>Issue type</label>
+				<div>
+					<p className="smallest">{rest.formik.values.requestType} > {rest.formik.values.subRequestType}</p>
+				</div>
+			</div>
+			<div className="address-search-wrapper">
+				<label htmlFor="location"
+					className="address">{pageFieldName.map(name => name.AddressHeaderLabel)}
+				</label>
+				<div className="address-input-wrapper">
+					<AutoCompletTypeField
+						items={items}
+						//name="location"
```

Comment:
@tmccoy529 @sgrg1 we should also start removing any commented code, we can always go back in history if we need to. There are some scenarios in which we might want to leave it, in those cases please add a comment and an issue so that we can resolve that work.